compiler/qsc_passes/src/entry_point.rs

```diff
@@ -124,13 +124,19 @@ fn create_entry_from_callables(
 fn get_callables(package: &Package) -> Vec<(&CallableDecl, LocalItemId)> {
     let mut finder = EntryPointFinder {
         callables: Vec::new(),
+        main: Vec::new(),
     };
     finder.visit_package(package);
-    finder.callables
+    if finder.callables.is_empty() {
+        finder.main
+    } else {
```

Comment:
If you have something marked as `@EntryPoint` and a main operation, should we issue a lint?

Reply:
Maybe... that could be considered for follow up. Today if you have an `@EntryPoint` and an entry expression nothing gets flagged. But this is a little more static than that. So I could see a lint warning about ambiguous entry point if there is a callable named `Main` and a separate callable with the attribute.

Reply:
Added it as an idea to #1333